qa/tasks/vstart_runner.py

```diff
@@ -773,6 +773,28 @@ def list_connections():
 
         self.gather_mount_info()
 
+    def _set_fuse_daemon_pid(self):
+        # NOTE: When a command <args> is launched with sudo, two processes are
+        # launched, one with sudo in <args> and other without. Make sure we
+        # get the PID of latter one.
+        for i in range(15):
```

Comment:
Better would be `teuthology.contextutil.safe_while`.

Reply:
Agree. I'll rewrite this.

Reply:
Added. Thank for pointing.